lib/js/components/Buttons/Button/Button.vue

```diff
@@ -21,6 +21,7 @@
 			'-elevation-x-small': elevation === ELEVATIONS.X_SMALL,
 			'-elevation-small': elevation === ELEVATIONS.SMALL,
 		}"
+		@click="$emit('click')"
```

Comment:
this makes it easier to bind to `click` event. We don't need to add `.native` modifier anymore